### PR TITLE
Add an alert when prow was not bumped for some time

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  prometheusAlerts+:: {
+    local comps = $._config.components,
```
Comment (cjwagner) [inline config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet:3]: This is unused.

**Review by cjwagner — APPROVED** [review]

File: config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    groups+: [
+      {
+        name: 'prow-stale',
+        rules: [
+          {
+            alert: 'Prow images are stale',
+            expr: |||
+              time()-max(prow_version) > %d * 24 * 3600
```
[the remark this answers is not in the document]
Comment (chaodaiG) [reply config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet:10]: I have tried `min` but got a value of 0, so use `max` to minimize noise, while still ensures it does check for staleness of bumps. Ensure version skew can be done by a separate rule imo.

File: config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet (hunk 1 above)
Comment (chaodaiG) [reply config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet]: Removed

File: config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet (hunk 2 above)
[the remark this answers is not in the document]
Comment (cjwagner) [reply config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet:10]: Ok. We might want to try to figure out where the 0 value came from though. That sounds like a bug, the version should be -1 or something large, right?

**Review by cjwagner — APPROVED** [review]

**Pull request merged**